### PR TITLE
RTIR: fix ID header bug

### PR DESCRIPTION
**Pull request by jochman**
First, need to merge: https://github.com/demisto/content/pull/7452
Then can review this pr
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/25429

## Description
Fix problem where there was an `ID` as a substring as a header.

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by Itay4** [comment]
lets also add a test for the given case, and verify it is not caught

File: Packs/RTIR/Integrations/RTIR/RTIR.yml [hunk 1]
```diff
@@ -585,3 +585,5 @@ script:
   subtype: python2
 tests:
 - RTIR Test
+fromversion: 4.1.0
+fromversion: 4.1.0
```
Comment (Itay4) [inline Packs/RTIR/Integrations/RTIR/RTIR.yml:589]: i think one is enough

File: Packs/RTIR/ReleaseNotes/1_0_2.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+- __RTIR__
+Lint and format fixes
```
Comment (Itay4) [inline Packs/RTIR/ReleaseNotes/1_0_2.md:4]: lets update to the actual rn here

File: Packs/RTIR/Integrations/RTIR/RTIR.yml (hunk 1 above)
Comment (jochman) [reply Packs/RTIR/Integrations/RTIR/RTIR.yml:589]: dsdk bug

**Pull request merged**